### ISSUE TITLE
allow for searching within groups and across groups

### ISSUE THREAD

**Issue by freakabcd** [photodiode/panorama-view#78]
I'm not sure about others, but I have used the search facilities of the Tab Groups addon extensively. Please implement a similar feature within the panorama view addon.

**Comment by tshatrov** [comment]
I also used the search feature a lot. It's good to have a hundred tabs open, but it's very difficult to find the tab I need.

**Comment by projectdelphai** [comment]
This project is currently abandoned/not maintained and I can't currently get access to this project to make any changes or fix any issues. If you need any help and want me to fix any issues or add any features, create an issue in my fork of this project [here](https://github.com/projectdelphai/panorama-tab-groups)!

That sounds like a good idea and I'll try and make it happen! Do you know if the search feature search the tab titles? or something further? If it's just the website title, it shouldn't be too hard to implement

**Comment by tshatrov** [comment]
Yeah, I think it only searched the tab title. You'd just type something in Panorama view and the matching tabs would get highlighted.

**Comment by photodiode** [comment]
Is this really an important feature?

It seems to me that searching with % in the URL field would be good enough.
I get the appeal of having the resulting tabs be highlighted, but implementing and maintaining a separate search system just for that seems like more trouble than it's worth.

Any thoughts?
